tests/fixtures/StyledComponent.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  $$typeof: Symbol('react.forward_ref'),
```

Comment:
What about checking this instead (check for ref forwarding)?

Reply:
I can, and it'll work (that was my original implementation) it just looked more mysterious that looking for `styledComponentId`. I'll change it to check for ref forwarding.

Reply:
Yeah, the problem with `styledComponentId` is that the implementation explicitly handles a case when a styled component is passed. But any other library that works similarly (but isn't `styled-components`) would fail to work. It's better to account for the most generic case.